Sources/SnapshotTesting/AssertSnapshot.swift

```diff
@@ -37,15 +53,16 @@ public func assertSnapshot<A, B>(
       identifier = String(counter)
     }
 
+    let testName = sanitizePathComponent(testName)
     let snapshotFileUrl = snapshotDirectoryUrl
-      .appendingPathComponent("\(testName.dropLast(2)).\(identifier)")
```

Comment:
This assumed the implicit `#function` format like `testName()`. Added a proper slugging function instead.